part2/docker-compose.yml

```diff
@@ -1,23 +1,21 @@
 version: '3.2'
-
 services:
   metastore:
     build:
       context: ./metastore
-    hostname: metastore
+    hostname: localhost
```

Comment:
this is working using podman-compose version  0.1.11. Could not test it with docker though (I have no docker installation)

Reply:
Obsolete, see later commits. I tested in podman and docker with the latest version.